dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/identifiable/IdentifiableServiceImpl.java

```diff
@@ -241,9 +241,11 @@ public I save(I identifiable) throws IdentifiableServiceException, ValidationExc
       return savedIdentifiable;
     } catch (CudamiServiceException e) {
       LOGGER.error(String.format("Cannot save UrlAliases for: %s", identifiable), e);
+      e.printStackTrace();
```

Comment:
und 248: Sollen die noch drin bleiben? Was passiert da in PROD (wo schreibt's das hin)?

Reply:
War nicht beabsichtigt, ist entfernt :)